smk/binning_preparation.smk

```diff
@@ -130,13 +130,13 @@ elif config['MEGAHIT_custom'] is not None:
             else:
                 config['MEGAHIT_presets'].append(f'meta-custom-{i+1}')
 
-batch_size = 100
-if config['CONTIG_MAPPING_BATCH_SIZE'] is None:
-    print('ERROR in ', config_path, ': CONTIG_MAPPING_BATCH_SIZE variable is empty. Please, complete ', config_path)
-elif type(config['CONTIG_MAPPING_BATCH_SIZE']) != int:
-    print('ERROR in ', config_path, ': CONTIG_MAPPING_BATCH_SIZE variable is not an integer. Please, complete ', config_path)
+max_job_ram_gb = 180
+if config['MAX_RAM_GB_AVAIL'] is None:
```

Comment:
Could we change it to something like `MAX_RAM_GB_PER_JOB` to be more specific? Otherwise, it could be tempting to write available RAM for the server, which will run one job at a time, which we don't really want.

Reply:
Done in https://github.com/arumugamlab/MIntO/pull/74/commits/510dd592fc827ce1c150485f2ca23952dcd77617